.pipelines/cni/singletenancy/linux-cniv2-template.yaml

```
@@ -71,7 +80,11 @@ stages:
                   kubectl apply -f config-reconcile.yaml
                   cd ../../../..
                   kubectl get po -owide -A
-                  sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(cnsVersion) CNI_DROPGZ_VERSION=$(dropgzVersion) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true TEST_DROPGZ=${{ parameters.testDropgz }}
+                  if [ "${{parameters.os}}" == "windows" ]; then
+                    sudo -E env "PATH=$PATH" make test-load CNSONLY=true CNS_VERSION=$(cnsVersion) CNI_DROPGZ_VERSION=$(dropgzVersion) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true TEST_DROPGZ=${{ parameters.testDropgz }}
```

nit: If changed, do change everywhere.
```suggestion
                    sudo -E env "PATH=$PATH" make test-load CNS_ONLY=true CNS_VERSION=$(cnsVersion) CNI_DROPGZ_VERSION=$(dropgzVersion) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true TEST_DROPGZ=${{ parameters.testDropgz }}
```